data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,615 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	"Greetings, human. To be of assistance to you, how can I?" the clerk says.`
+			`	Anna speaks up. "I'm Anna Fangvin, and I'd like to publish a book here in Coalition space. My friend <first> <last> tells me I will need a translation box to converse with the publishing company. Could you provide one?"`
+			`	"A translator, the human needs?" asks the clerk.`
+			choice
+				`	"Yes."`
+			`	"Trust you with a translator, we cannot. And help you, I cannot, so a Heliarch, I must call," says the clerk, then proceeding to call for a Heliarch in its native language.`
+			`	After a few minutes, another Kimek wearing the telltale golden circlet of a Heliarch approaches you and directs you into a room.`
+			`	"Told me about your intentions, the clerk already has, and understand it, I do. Accompany you to the publishers, me and my agents will, once decided you have upon a company." The Arachi sits down.`
+			`	"We need to find a publishing company then!" says Anna.`
+			label questions
+			choice
+				`	"The Kimek might be a good choice. They have a high population."`
+					goto publishingcompany
+				`	"Perhaps the Saryd. People with human parts are likely to be more interested in human books."`
+					goto saryd
+					to display
+						not "struggling author: saryd"
+				`	"The Arachi have a close-knit population. Word of a new book might spread quickly there."`
+					to display
+						not "struggling author: arachi"
+			action
+				set "struggling author: arachi"
+			`	"Oh, it's kind of embarrassing but... I'm actually a bit afraid of spiders, so I'm going to take a while to get used to the Arachi."`
+				goto questions
+			label saryd
+			action
+				set "struggling author: saryd"
+			`	"I thought the same at first, but from what I've seen the Saryds seem pretty reserved, don't you think? Maybe another species would be better."`
+				goto questions
+			label publishingcompany
+			action
+				clear "struggling author: arachi"
+				clear "struggling author: saryd"
+			`	"But where can we find a Kimek publishing company? We haven't even translated our book into their language yet!"`
+			choice
+				`	"We'll find a company."`
+				`	"Given the number of books here, there has to be a publishing company somewhere."`
+			`	"The faster we can get the book published, the better. Meet me in the spaceport in a few hours; I'll find a good company."`
+			`	Before Anna can head off, the Heliarch rises. "Accompany you, I will. Suitable for two aliens to roam around our space unattended, it is not."`
+
+
+mission "Coalition: Struggling Author 4"
+	name "Kimek Publishing Company"
+	description "Anna found a Kimek publishing company who is willing to take in Anna Fangvin to get her book published. Bring Anna Fangvin to <destination>."
+	source "Ki Patek Ka"
+	destination "Inmost Blue"
+	blocked "You do not have enough bunk space for the Heliarch agent and Anna Fangvin. Come back here when you have two bunks free."
```

Comment:
```suggestion
	blocked "You do not have enough bunk space for Anna Fangvin and the Heliarch agent. Come back here when you have two bunks free."
```

Probably should put Anna's name first, as she is more relevant to the storyline than the Heliarch agent.